### PR TITLE
SDK update 2.1.0, fixed random fails

### PR DESCRIPTION
**Pull request by OlehVeselov92**

### CODE REVIEW

File: src/tools/api.js [hunk 1]
```diff
@@ -16,53 +17,11 @@ const { API_KEY, UUID } = process.env;
...
-                        typeaheadId: Date.now(),
-                        uiOrigin: 'SINGLE_SEARCH_HERO',
-                    },
-                    articleCategories: [],
-                    enabledFeatures: ['typeahead-q'],
-                },
-            },
-        }],
+        body: [{ query: SearchQuery,
+            variables: { request: { query, limit: 10, scope: 'WORLDWIDE', locale: 'en-US', scopeGeoId: 1, searchCenter: null, types: ['LOCATION', 'QUERY_SUGGESTION', 'USER_PROFILE', 'RESCUE_RESULT'], locationTypes: ['GEO', 'AIRPORT', 'ACCOMMODATION', 'ATTRACTION', 'ATTRACTION_PRODUCT', 'EATERY', 'NEIGHBORHOOD', 'AIRLINE', 'SHOPPING', 'UNIVERSITY', 'GENERAL_HOSPITAL', 'PORT', 'FERRY', 'CORPORATION', 'VACATION_RENTAL', 'SHIP', 'CRUISE_LINE', 'CAR_RENTAL_OFFICE'], userId: null, context: { searchSessionId: '91CDC537DA7536533B97CF5EEFE80DA41638306855528ssid', typeaheadId: '1638306877219', uiOrigin: 'SINGLE_SEARCH_HERO', routeUid: '20b035aa-a83c-4750-ad15-234206bd6c5d' }, articleCategories: ['default', 'love_your_local', 'insurance_lander'], enabledFeatures: ['typeahead-q'] } } }],
```
Comment (metalwarrior665) [inline src/tools/api.js]: Maybe just put this to separate file so it doesn't create this super long row. It is a constant anyway.

**Review by metalwarrior665 — CHANGES_REQUESTED** [review]
Thanks, looks good

File: src/tools/api.js (hunk 1 above)
Comment (OlehVeselov92) [reply src/tools/api.js]: "query" is dynamic there.
I will format it like a normal object.
Looks good for me in this way)
Or shall I still make a separate file for it?

Comment (metalwarrior665) [reply src/tools/api.js]: Looks good now

**Review by metalwarrior665 — APPROVED** [review]

**Pull request merged**